labelCloud/view/gui.py

```diff
@@ -187,6 +191,22 @@ def __init__(self, control: "Controller") -> None:
         self.timer.timeout.connect(self.controller.loop_gui)
         self.timer.start()
 
+    #open 2D(png) image
+    def open_2D_img(self):
+        tmp_cur_file = str(self.controller.pcd_manager.get_current_name().split(".")[0])
+        filename = r'C:/Users/user/Desktop/code_cloud/2d_button_cloud/pointclouds/' + tmp_cur_file + '.png'
+
+        if os.path.isfile(filename):
+            image = QtGui.QImage(QtGui.QImageReader(filename).read())
+            self.imageLabel = QLabel()
+            self.imageLabel.setWindowTitle("2D image")
+            self.imageLabel.setPixmap(QPixmap.fromImage(image))
+            self.imageLabel.show()
+
+        else:
+            QMessageBox.information(self, 'No 2d image File', "There's no " + tmp_cur_file + 
```

Comment:
Maybe use the whole path to ease debugging.

Reply:
I'm not sure whether I'm with you or not, I modified a bit but I'm wondering if you want to show users a whole path like using os
ex:

```import os
os.path.abspath(os.getcwd())

Reply:
My thought is that new users might not know were labelCloud is looking for the images. So printing the whole path to the file (like it is currently done) seems to help more.